client/components/foldable-card/index.jsx

```diff
@@ -86,9 +87,11 @@ var FoldableCard = React.createClass( {
 		}
 		if ( this.props.children ) {
 			let iconSize = 24;
+			let iconDefaultText = ( typeof this.translate === 'function' ) ? this.translate( 'More' ) : '';
+			let iconText = this.props.iconText || iconDefaultText;
 			return (
 				<button disabled={ this.props.disabled } className="foldable-card__action foldable-card__expand" onClick={ clickAction }>
-					<span className="screen-reader-text">{ this.translate( 'More' ) }</span>
+					<span className="screen-reader-text">{ iconText }</span>
```

Comment:
Fixed in https://github.com/Automattic/wp-calypso/pull/4297/commits/75a286085c8e2adc9086b148f6b47adaba594da2